src/v/cluster/node/local_monitor.cc

```diff
@@ -45,15 +44,15 @@ ss::future<> local_monitor::update_state() {
 
 const local_state& local_monitor::get_state_cached() { return _state; }
 
-// TODO make async
-std::vector<disk> local_monitor::get_disks() {
-    auto space_info = std::filesystem::space(
-      config::node().data_directory().path);
+ss::future<std::vector<disk>> local_monitor::get_disks() {
+    auto svfs = co_await ss::engine().statvfs(
```

Comment:
@dotnwat note this still uses statvfs() directly. Wondering if you had any thoughts on my [comment](https://github.com/vectorizedio/redpanda/pull/3414#discussion_r780601014) response in the draft PR. Happy to change to multiple calls to the public API if you like (would probably just add an additional patch versus rebasing again).

Reply:
I noticed that we also grab this sort of state in `syschecks.cc`, so the nice thing to do would probably be to have both the syschecks and the local health checks use the same codepath. Let's leave it for now, but I'll be responsible to fix it if the API diverges.